doc/en/source/k8sIntegration.rst

```diff
@@ -0,0 +1,186 @@
+**********************
+Kubernetes Integration
+**********************
+
+The components of FogFlow can be built via source code as well as in docker environment using docker-compose tool. In docker environment each component of FogFlow is running on a single container. Whole FogFlow system will have to re-start in case any single component container goes down and if any single service is overloaded it cannot scale to handle the load.  
```

Comment:
change " In docker environment each component of FogFlow is running on a single container. " to  In docker environment each component of FogFlow is running running as single instance.